capstone/src/main/webapp/lesson-loader.js

```diff
@@ -70,55 +70,80 @@ function createContent(title, description, content, urls) {
     });
 }
 
-function getLessons() {
-    fetch(queryString, {method: "GET", headers: new Headers({ID_TOKEN})}).then(response => response.json()).then((lessonsList) => {
-        const lessonElement = document.getElementById("lesson-container");
-        lessonElement.innerHTML = "";
-        for (lesson of lessonsList) {
-            lessonElement.appendChild(createLessonDivElement(lesson));
-        };
-    });
-}
-
 function joinRoom() {
-    console.log("Room joined!");
     return fetch("/join?room_id=" + getRoomId() + "&action=join", {method: "POST", headers: new Headers({ID_TOKEN})}); 
 }
 
 function unjoinRoom() {
-    console.log("User has left the room!");
-    return fetch("/join?room_id=" + getRoomId() + "&action=unjoin",
-    {method: "POST", headers: new Headers({ID_TOKEN})});
+    return fetch("/join?room_id=" + getRoomId() + "&action=unjoin", {method: "POST", headers: new Headers({ID_TOKEN})});
 }
 
 function getRoomId() {
     return new URL(window.location.href).searchParams.get("room_id");
 }
 
 function getJoinStatus() {
-    var action = new URL(window.location.href).searchParams.get("action")
+    var action = new URL(window.location.href).searchParams.get("action");
     if (action == null) {
         return "unjoin";
     }
```

Comment:
This is another instance where you can use the nullish operator to check for null and apply a default:

return new URL(window.location.href).searchParams.get("action") ?? "unjoin";